pkg/printers/internalversion/describe.go

```diff
@@ -2875,6 +2875,24 @@ func describeHorizontalPodAutoscaler(hpa *autoscaling.HorizontalPodAutoscaler, e
 				w.Write(LEVEL_0, "failed to check Replication Controller\n")
 			}
 		}
+		if strings.ToLower(hpa.Spec.ScaleTargetRef.Kind) == "replicaSet" {
```

Comment:
ToLower will never match "replicaSet"

We should not be adding hard coded scale targets to this function. If we want to use the generic scale client to fetch scale status, that would be better